libs/driver/magento/src/graphql/apollo-cached-operation-converter.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { DaffMagentoApolloCachedOperationConverter } from '@daffodil/driver/magento';
+
+describe('Driver | Magento | GraphQl | DaffMagentoApolloCachedOperationConverter', () => {
+	
+	it('should set the operation to a GET method', () => {
+		const mockOperation = {
+			query: null,
+			operationName: null,
+			extensions: null,
+			variables: null,
+			setContext: jasmine.createSpy(),
```

Comment:
How difficult would it be to create a real operation object and assert the HTTP method is `'GET'`?

Reply:
It's a little janky, but better than the spy. Done

Reply:
did you forget to push it? I don't see it

Reply:
I don't think so

Reply:
LUL

I like this test much better though even though its more integration than unit.